espnet2/tasks/enh.py

```diff
@@ -187,7 +189,74 @@ def add_task_arguments(cls, parser: argparse.ArgumentParser):
             "--use_preprocessor",
             type=str2bool,
             default=False,
-            help="Apply preprocessing to data or not",
+            help="Whether to apply preprocessing to data or not",
+        )
+        group.add_argument(
+            "--speech_volume_normalize",
+            type=float_or_none,
+            default=None,
+            help="Scale the maximum amplitude to the given value.",
+        )
+        group.add_argument(
+            "--rir_scp",
+            type=str_or_none,
+            default=None,
+            help="The file path of rir scp file.",
+        )
+        group.add_argument(
+            "--rir_apply_prob",
+            type=float,
+            default=1.0,
+            help="THe probability for applying RIR convolution.",
+        )
+        group.add_argument(
+            "--noise_scp",
+            type=str_or_none,
+            default=None,
+            help="The file path of noise scp file.",
+        )
+        group.add_argument(
+            "--noise_apply_prob",
+            type=float,
+            default=1.0,
+            help="The probability applying Noise adding.",
+        )
+        group.add_argument(
+            "--noise_db_range",
```

Comment:
isn't it better to use SNR ? Because if speech material level varies a lot, then for some utterance you can be unlucky and end up with very low (or very high) SNR. If you sample instead the SNR you have more control.

Reply:
I think it is actually SNR (see [here](https://github.com/espnet/espnet/pull/4321/files#diff-bf4e9d668125c9efceb4d3a18b2f2df71d95f59596d0b442cb5aee2d9d270b05R260-R266)), although the name is a bit confusing.

Reply:
I just kept the argument name as it was introduced in ASR preprocessors.

Reply:
Ok thanks, makes sense ! maybe specify it in the help.